src/app/common/authentication/use-finish-auth-request.ts

```diff
@@ -58,27 +42,7 @@ export function useFinishAuthRequest() {
 
       // We can't perform any of this logic for non-software wallets
       // as they require the key to be available in the JS context
```

Comment:
We have a type for this - `type WalletType = 'ledger' | 'software';`

We could change that to an `enum` and not repeat the logic - ` walletType === 'software'`

Instead using ` walletType === WalletType.Software`

Reply:
I'm often torn between string literals and enums. 

When typing `walletType === '` typescript will autocomplete the string, whereas `WalletType` requires an extra import.